documentation/source/newsfragments/1489.removal.rst

```diff
@@ -0,0 +1 @@
+:class:`ReturnValue` is now deprecated. It will be removed in the release following 1.4. Use a :keyword:`return` statement instead; this works in all supported versions of Python.
```

Comment:
```suggestion
:class:`ReturnValue` is now deprecated. Use a :keyword:`return` statement instead; this works in all supported versions of Python.
```

Let's not make promises about when things will be removed. The fact it is deprecated indicates that we will remove it without warning at some point.